### DISCUSSION TITLE
Can't open files from NAS

### DISCUSSION THREAD

**Discussion by lsahnicne** [freeplane/freeplane#1426] [Help]
I store a mind map file on my NAS. NAS remote directory is mounted in /mnt/ directory on my workstation. When i try to open the file in Freeplane, the dialogue window doesn't display /mnt/ directory. How can I access a file that is not on a local machine, but on a remote host?

![Screenshot from 2023-09-24 10-31-24](https://github.com/freeplane/freeplane/assets/43319094/24878344-c2b7-40e4-9636-3ca5c3dedc71)

**Comment by macmarrum** [comment]
It looks like an issue with your permissions rather than with Freeplane, similar to
- https://github.com/freeplane/freeplane/issues/1286
